lib/msal-browser/src/config/Configuration.ts

```diff
@@ -111,19 +117,20 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
     };
 
     // Default cache options for browser
-    const DEFAULT_CACHE_OPTIONS: CacheOptions = {
+    const DEFAULT_CACHE_OPTIONS: Required<CacheOptions> = {
         cacheLocation: BrowserCacheLocation.SessionStorage,
         storeAuthStateInCookie: false
     };
 
     // Default logger options for browser
     const DEFAULT_LOGGER_OPTIONS: LoggerOptions = {
         loggerCallback: (): void => {},
+        logLevel: LogLevel.Info,
         piiLoggingEnabled: false
     };
 
     // Default system options for browser
-    const DEFAULT_BROWSER_SYSTEM_OPTIONS: BrowserSystemOptions = {
+    const DEFAULT_BROWSER_SYSTEM_OPTIONS: Required<Omit<BrowserSystemOptions, "loadFrameTimeout">> = {
```

Comment:
Why is `loadFrameTimeout` omitted?

Reply:
The only place it's used is to set the `windowHashTimeout` and `iframeHashTimeout` below, it's not needed anywhere else.

Reply:
Could the default by set to `0` so that it doesn't need to be omitted?

Reply:
Done